build.gradle.kts

```diff
@@ -17,15 +17,32 @@ liquibase {
         register("main") {
             arguments = mapOf(
                 "changeLogFile" to "db/db.changelog-master.xml",
-                "url" to databaseCredentials.databaseUrl,
+                "url" to "${ databaseCredentials.databaseUrl}?createDatabaseIfNotExist=true",
```

Comment:
Does `databaseUrl` still contain schema name here? Or will it be overridden by `liquibaseSchemaName`?